### DISCUSSION TITLE
Unable to find elements in the browser window which is opened by Electron desktop application

### DISCUSSION THREAD

**Discussion by Siva1212Copado** [webdriverio/webdriverio#13510] [Technical Discussion]
Hi @christian-bromann 
Can you please help out for below example how to automate scenario 
Example: 
I logged into the electron desktop application and click signin button then user redirected to sign in page in the system deafultfault browser when I am writing code the automate the elements which are present in the signin page(browser) elements were not identified by webdriver io and tests are failing ,
Can you please help me out how to change to focus to browser page from the desktop application ?
I tried to use get windowhandles and swtictowindow handle method were not working

**Comment by christian-bromann** [comment]
@Siva1212Copado unfortunately you can't automate this scenario very easily. WebdriverIO can only automate what's going on in the application and can't "switch" to automate the browser. There are certainly ways to automate multiple OS apps and I would recommend to look into Appium for doing that.
